tests/generic_config_updater/gu_common_test.py

```diff
@@ -675,6 +675,20 @@ def test_find_ref_paths__whole_config_path__returns_all_refs(self):
         # Assert
         self.assertEqual(expected, actual)
 
+    def test_find_ref_paths__path_and_ref_paths_are_under_same_yang_container__returns_ref_paths(self):
+        # Arrange
+        path = "/LOOPBACK_INTERFACE/Loopback0"
+        expected = [
+            "/LOOPBACK_INTERFACE/Loopback0|10.1.0.32~132",
```

Comment:
>~1 [](http://example.com/codeflow?start=52&length=2)

To improve readability regarding the `~1` escape, we con construct the paths from components.

eg: [https://github.com/Azure/sonic-mgmt/blob/d23d893a40e898fb3aeb70abd3bd60d08492b484/tests/generic_config_updater/gu_utils.py#L244](https://github.com/Azure/sonic-mgmt/blob/d23d893a40e898fb3aeb70abd3bd60d08492b484/tests/generic_config_updater/gu_utils.py#L244)

Reply:
updated